docs/aggregation.rst

```diff
@@ -37,6 +37,15 @@ Meridional Overturning Circulation Basins
 The function :py:func:`geometric_features.aggregation.ocean.moc` aggregates
 basins for computing the MOC: Atlantic, IndoPacific, Pacific and Indian
 
+Arctic Regions
+~~~~~~~~~~~~~~
+
+The function :py:func:`geometric_features.aggregation.ocean.arctic` aggregates
+regions of the Arctic: Baffin Bay, Baltic Sea, Barents Sea, Canada Basin,
```

Comment:
Above, you may want to modify to `of the Arcic (and subArctic)`.

Reply:
Could add: 'Arctic Ocean (no Barents/Kara Seas)' (or related name), 'Beaufort Gyre', and 'Beaufort Gyre shelf'.

Reply:
@xylar: don't forget to modify this text above before merging.

Reply:
Thanks, I missed these suggestions somehow.